homeassistant/components/suez_water/quality_scale.yaml

```diff
@@ -0,0 +1,88 @@
+rules:
+  # Bronze
+  config-flow: todo
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: todo
+  runtime-data:
+    status: todo
+    comment: coordinator is created during setup, should be stored in runtime_data
+  test-before-setup: done
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: done
+  entity-event-setup:
+    status: exempt
+    comment: no subscription to api
+  dependency-transparency: done
+  action-setup:
+    status: exempt
+    comment: no service action
+  common-modules: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  docs-actions:
+    status: exempt
+    comment: no service action
+  brands: done
+
+  # Silver
+  config-entry-unloading: done
+  log-when-unavailable: todo
+  entity-unavailable: done
+  action-exceptions:
+    status: exempt
+    comment: no service action
+  reauthentication-flow: todo
+  parallel-updates:
+    status: exempt
+    comment: no service action and coordinator updates
+  test-coverage: done
+  integration-owner: done
+  docs-installation-parameters:
+    status: todo
+    comment: missing user/password
+  docs-configuration-parameters:
+    status: exempt
+    comment: no configuration option
+
+  # Gold
+  entity-translations: todo
+  entity-device-class: done
+  devices: done
+  entity-category: todo
+  entity-disabled-by-default: todo
+  discovery:
+    status: exempt
+    comment: api only, nothing on local network to discover services
```

Comment:
How does the meter connect?

Reply:
Unfortunately nothing goes through local network.

From the provider [website](https://www.suez.com/en/water/sustainable-practices/water-consumption/remote-reading-water-meter)

> Water meters equipped with transmitters communicate their consumption data daily via radio transmission to receivers, which in turn transmits them to data processing software. The data is then made available on the platforms of operators and users.

Reply:
Ah check! Thanks!